src/app/services/api.service.ts

```diff
@@ -12,10 +12,10 @@ export abstract class ApiService<T extends ResourceModel<T>> {
 
   private baseUrl = String(environment.API_URL);
 
-  public create(resource: Partial<T> & { toJson: () => T }): Observable<any> {
```

Comment:
api service is super common fiture. Please verify that you didn't break anything by these changes before the merge.